yt_dlp/extractor/twitter.py

```diff
@@ -1585,6 +1587,10 @@ def _real_extract(self, url):
         if not broadcast:
             raise ExtractorError('Broadcast no longer exists', expected=True)
         info = self._parse_broadcast_data(broadcast, broadcast_id)
+        info['title'] = broadcast.get('status') or info.get('title')
```

Comment:
can you add a test that has a title actually sourced from `broadcast['status']`?

Reply:
On second look...

The Periscope base extractor sets the `info['title']` value to `broadcast['status']` prefixed with `{uploader} - `, which is consistent with the `TwitterIE` title scheme. 

So I'm not sure if this is a necessary or desired change

Reply:
Twitter broadcast has its own title, so I think is it better for `title` to exclude `uploader` info

Reply:
if it's null you are falling back to what the periscope base extractor sets anyways, so there's no difference

Reply:
I just don't see the justification for changing this when it is already in line with the main twitter extractor

Reply:
Hmm I mean currently, if I run this

```
yt-dlp https://twitter.com/i/broadcasts/1ZkKzeyrPbaxv --print title
```

It will return the title as `SpaceX - Starship | SN10 | High-Altitude Flight Test`

But to combine with format, I usallly have `--output "~/[%(uploader)s][%(upload_date)s] %(title)s (%(id)s).%(ext)s"`

Which the title repeat `uploader` but it not necessary here